README.md

```diff
@@ -93,7 +93,7 @@ Eg: `curl ... | NVM_DIR="path/to/nvm"`. Ensure that the `NVM_DIR` does not conta
 
 #### Troubleshooting on Linux
 
-On Linux, after running the install script, if you get `nvm: command not found` or see no feedback from your terminal after you type `command -v nvm`, simply close your current terminal, open a new terminal, and try verifying again.
+On Linux, after running the install script, if you get `nvm: command not found` or see no feedback from your terminal after you type `command nvm -v`, simply close your current terminal, open a new terminal, and try verifying again.
```

Comment:
this is incorrect - the goal isn't to run `nvm -v` with `command`, it's to run `command -v` with `nvm`.